packages/path_provider/path_provider_android/test/path_provider_android_test.dart

```diff
@@ -89,12 +89,6 @@ void main() {
       });
     } // end of for-loop
 
-    test('getExternalStoragePaths with null android succeeds', () async {
```

Comment:
This test was asserting bad behavior, null should have been passed down to Android.